pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -228,10 +228,10 @@ func (m *kubeGenericRuntimeManager) startContainer(podSandboxID string, podSandb
 		if handlerErr != nil {
 			m.recordContainerEvent(pod, container, kubeContainerID.ID, v1.EventTypeWarning, events.FailedPostStartHook, msg)
 			if err := m.killContainer(pod, kubeContainerID, container.Name, "FailedPostStartHook", reasonFailedPostStartHook, nil); err != nil {
-				klog.ErrorS(fmt.Errorf("%s: %v", ErrPostStartHook, handlerErr), "Failed to kill container", "pod", klog.KObj(pod),
+				klog.ErrorS(err, "Failed to kill container", "pod", klog.KObj(pod),
```

Comment:
With ``ErrPostStartHook``  symbol may help us filter 'ErrPostStartHook'  type error  quickly.

Reply:
So we should keep the log as before.

It would simply revert #54739 which fixed #54671.

Reply:
Updated @dabaooline